modules/jenkins.test.js

```diff
@@ -0,0 +1,111 @@
+// -----------------------------------------------------------------------------------------
+// #region Imports
+// -----------------------------------------------------------------------------------------
+
+const jenkins = require("./jenkins");
+const fs = require("fs");
+
+// #endregion Imports
+
+// -----------------------------------------------------------------------------------------
+// #region Tests
+// -----------------------------------------------------------------------------------------
+
+describe("jenkins", () => {
+    describe("configureCredentials", () => {
+        it("when url is empty then returns false", () => {
+            // Arrange
+            const url = "";
+            const username = "aaa";
+            const token = "aaa";
+            // Act
+            const result = jenkins.configureCredentials(url, username, token);
+            // Assert
+            expect(result).toBe(false);
+        });
+
+        it("when username is empty then returns false", () => {
+            // Arrange
+            const url = "aaa";
+            const username = "";
+            const token = "aaa";
+            // Act
+            const result = jenkins.configureCredentials(url, username, token);
+            // Assert
+            expect(result).toBe(false);
+        });
+
+        it("when token is empty then returns with errors", () => {
+            // Arrange
+            const url = "aaa";
+            const username = "aaa";
+            const token = "";
+            // Act
+            const result = jenkins.configureCredentials(url, username, token);
+            // Assert
+            expect(result).toBe(false);
+        });
+
+        it("when url, username and token provided then returns true ", () => {
```

Comment:
Super nitpicky - I think we've been using `test()` across the board for this project, can we swap these out for consistency?